src/AppBundle/Helper/ApiHelper.php

```diff
@@ -50,6 +51,7 @@ public function groups($project, $username)
             if (isset($res["batchcomplete"]) && isset($res["query"]["users"][0]["groups"])) {
                 $result = $res["query"]["users"][0]["groups"];
             }
+            $this->cacheSave($cacheKey, $result, 'P7D');
```

Comment:
This info along with global user groups indeed rarely change, but unlike the namespaces this same info may not be requested over and over. Given how popular the tool is, is it worth caching that info in Redis for that long?